homework-submission/todo-app/src/readme.md

```diff
@@ -0,0 +1,28 @@
+// Counter practice
+```
```

Comment:
This addition doesn't seem very relevant for the homework. Could you remove that? Makes it easier to review :-)

Reply:
removed.

Reply:
I added this because in the class my pc wasn't running good, and class practice was failed.